src/erl_csv_decoder.erl

```diff
@@ -64,33 +68,38 @@ get_more_stream(Stream) ->
             NewStream
     end.
 
-% -spec process_match({error, term()} | match(), csv_decoder(), binary()) ->
-%     {ok, binary()} | {has_trailer, iolist(), binary()} | {nomatch, binary()} | {error, term()}.
+-spec process_match(
+        {match, matches()} | {error, term()} | match | nomatch, csv_decoder(), iodata()) ->
```

Comment:
The `match` atom does not make sense to me...

Reply:
It's on the return values of the `re:run`, dialyzer and eqwalizer forced me to add that value here before they'd stop complaining ☹️